Admin/AbstractAdmin.php

```diff
@@ -66,6 +66,7 @@
         )\\\(.*)@x';
 
     const MOSAIC_ICON_CLASS = 'fa fa-th-large fa-fw';
+    const DEFAULT_EXPORT_FORMATS = array('json', 'xml', 'csv', 'xls');
```

Comment:
Just read the diff...

You extracted a new constant which is only used once. So why did we need this?